libraries/backend/azure_connection.rb

```diff
@@ -157,7 +162,10 @@ def fail_api_query(resp, message = nil)
     message ||= "Unsuccessful HTTP request to Azure REST API.\n"
     message += "HTTP #{resp.status}.\n"
     body = resp.body
-    unless body.empty?
+    if body.empty?
```

Comment:
Is this change intentional? I would have imagined the message body can have content when successful also.

Reply:
This is in the `fail_api_query` method which is called only when the HTTP status is not 200.
It is just a refactoring to prevent increasing the conditional statement depth. The functionality is the same.

Reply:
cool no worries